package.json

```diff
@@ -33,6 +33,7 @@
     "jest-environment-jsdom": "29.6.3",
     "npm-check-updates": "16.12.2",
     "nyc": "15.1.0",
+    "prettier": "3.0.3",
```

Comment:
Can you help me understand this change?

Before, Prettier and ESLint were driven by `@medplum/eslint-config`, which has `prettier` as a `devDependency`.  Will that no longer be the case?

Reply:
I think there were some problems when it came to running `prettier` is VSCode, so I added it to the root of the project. I think some files were not being formatted or checked by `prettier` previously, so that's why we had so many conflicts in #2758

Reply:
We actually may be able to remove `prettier` from `@medplum/eslint-config` now since it's no longer running in `ESLint` after this PR...